polyfill/test/Calendar/prototype/fields/argument-throws-duplicate-keys.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.calendar.prototype.fields
+info: |
+    sec-temporal.calendar.prototype.fields step 7.b.iii:
+      iii. If _fieldNames_ contains _nextValue_, then
+        1. Let _completion_ be ThrowCompletion(a newly created *RangeError* object).
+        2. Return ? IteratorClose(_iteratorRecord_, _completion_).
+features: [Temporal]
+---*/
+
+const calendar = new Temporal.Calendar("iso8601");
+assert.throws(RangeError, () => calendar.fields(["day", "month", "day"]));
+assert.throws(RangeError, () => calendar.fields(["year", "month", "monthCode", "day", "hour", "minute", "second", "millisecond", "microsecond", "nanosecond", "year"]));
+
+const manyFields = {
+  count: 0
+};
+
+manyFields[Symbol.iterator] = function*() {
+  while (this.count++ < 100)
+  {yield "year";}
```

Comment:
Formatting is a bit weird here?
```suggestion
  while (this.count++ < 100) yield "year";
```

Reply:
Hm, I ran 'npm pretty' before pushing, pretty sure that's where this came from; I certainly didn't write it with the {..} like that.